### ISSUE TITLE
AppStoreConnect WARNING about MinimumOSVersion

### ISSUE THREAD

**Issue by alpha-moisol** [AppsFlyerSDK/AppsFlyerFramework#283]
# Report

## SDK Version
`6.14.0`

## What did you do?

Uploaded app build with AppsFlyer-Dynamic dependency to AppStoreConnect

## What did you expect to happen?

Build is uploaded without warnings

## What happened instead?
Got email from AppStoreConnect stated

```
Although delivery was successful, you may want to correct the following issues in your next delivery. Once you've corrected the issues, upload a new binary to App Store Connect.

ITMS-90747: Architecture incompatible with MinimumOSVersion. The app bundle at '***.app/Frameworks/AppsFlyerLib.framework' specifies a MinimumOSVersion of '100.0' but contains a 32-bit architecture that is unsupported on iOS 12 and later.
```

**Comment by bogdanmatasaru** [comment]
please fix this

**Comment by popwarfour** [comment]
Fix please. Your checksums are also now wrong on `6.14.0`

**Comment by PrakashT** [comment]
Hello, we are having the same issue, any update on this issue?

**Comment by kgaidis** [comment]
There are a bunch of other issues around this:
- https://github.com/AppsFlyerSDK/AppsFlyerFramework/issues?q=is%3Aissue+MinimumOSVersion

The only thing that worked for me was switching from "Dynamic" to "Static"  (the other solutions like clearing SPM cache were unsuccessful for me):

https://github.com/AppsFlyerSDK/AppsFlyerFramework-Static

**Comment by dandreiolteanu** [comment]
Same issue here, any plans to update this?

**Comment by TheCoordinator** [comment]
This seems to be quite critical, how come we're not hearing from AppsFlyer team?

**Comment by popwarfour** [comment]
https://github.com/googleads/swift-package-manager-google-mobile-ads/issues/62

The version 100 appears to be an Xcode 15.3 work around. It’s an annoying warning but seems pretty harmless for now. The dynamic lib is working for us at the moment though.

There is a separate, far worse issue, where the checksums defined in the package file are not matching the underlying binary lib. I haven’t dug into why this happened but I suspect the release was botched and not resolved properly vs a local SPM cache issue.

**Comment by skwashua** [comment]
The static library also fails to upload with Xcode 15.3, min target 16.4, using SPM

**Comment by ge-af** [comment]
Hi team, we released iOS SDK **v6.14.2** that should address the above.

Please let us know if you stumble upon any issue while upgrading to it. You can also reach out to our Support team at [support@appsflyer.com](mailto:support@appsflyer.com) and we will be more than happy to assist.
